pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "pycontw-backend"
+version = "0.1.0"
+description = "The API backend and proposal system of PyCon TW website."
+authors = ["PyCon TW Web Team <dev@python.tw>"]
+readme = "README.md"
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "~3.10.0"
+
+Django = "3.1.7"
+dj-registry = "0.2.2"
+django-cors-headers = "3.6.0"
+django-compressor = "2.4"
+django-compressor-toolkit = "0.6.0"
+django-crispy-forms = "1.8.1"
+django-environ = "0.11.2"
+django-extensions = "2.2.6"
+django-import-export = "2.0.1"
+django-modeltranslation = "0.16.2"
+django-storages = "1.11.1"
+djangorestframework = "3.12.2"
+google-cloud-storage = "1.36.1"
+ipython = "7.16.3"
+lxml = "4.6.3"
+psycopg2-binary = "2.8.5"
```

Comment:
**question:** according to [official documentation](https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary) and [PyPI](https://pypi.org/project/psycopg2-binary/),
> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

shall we put binary version in `dev.dependencies` and use `psycopg2==2.8.4` in production?

Reply:
Oh yeah, agree with you on that. Just for confirmation, is that meaning we could just install `psycopg2` instead of `psycopg2-binary` because it seems we are adopting `psycopg2-binary` in the `pyproject.toml`. Did I miss something?

Reply:
my bad, on it

Reply:
fixed